src/main/java/uk/ac/ebi/ena/app/menu/services/MenuService.java

```diff
@@ -364,6 +370,7 @@ private String eRequestEmailId(DownloadFormatEnum format, String location, Downl
         MenuUtils.printBackMessage();
 
         String input = scannerUtils.getNextString();
```

Comment:
why not wrap scannerUtils.getNextString() in trim directly? this applies to all places where you do scannerutils